src/api/Themer.ts

```diff
@@ -65,30 +65,28 @@ export function themerInit(constants: typeof Constants) {
 export function applyTheme() {
     try {
         if (currentTheme === undefined) return;
+        const colors = currentTheme.colors ?? currentTheme.colours;
         for (const key in currentTheme.theme_color_map) {
             if (!discordConstants.ThemeColorMap[key]) continue;
 
-            discordConstants.ThemeColorMap[key][ThemeType.AMOLED] = discordConstants.ThemeColorMap[key][ThemeType.DARK];
             if (currentTheme.theme_color_map[key]) {
-                discordConstants.ThemeColorMap[key][ThemeType.AMOLED] = currentTheme.theme_color_map[key]?.[ThemeType.DARK];
+                if (currentTheme.theme_color_map[key]?.[ThemeType.AMOLED]) discordConstants.ThemeColorMap[key][ThemeType.AMOLED] = currentTheme.theme_color_map[key]?.[ThemeType.AMOLED];
                 discordConstants.ThemeColorMap[key][ThemeType.LIGHT] = currentTheme.theme_color_map[key]?.[ThemeType.LIGHT];
                 discordConstants.ThemeColorMap[key][ThemeType.DARK] = currentTheme.theme_color_map[key]?.[ThemeType.DARK];
             }
         }
 
         // Enmity compat
```

Comment:
we should only do enmity compat if theme_color_map does not exist (aliu takes priority)